src/interactive-window/types.ts

```diff
@@ -46,6 +46,7 @@ export interface IInteractiveWindowProvider {
      * @param owner The URI of a text document which may be associated with an interactive window.
      */
     get(owner: Uri): IInteractiveWindow | undefined;
+    getActiveInteractiveWindow(): IInteractiveWindow | undefined;
```

Comment:
Isn't this a duplicate of the `activeWindow` property?

Reply:
Looking at the interface there doesn't seem to be any difference , except one is a property & the other (new code) is a method

Reply:
@DonJayamanne they are different in terms of implementation. `getActiveInteractiveWindow` checks `activeWindow` first, and then check `activeTextEditor`  and try to find the interactive window through resource uri mapping.

The name `active` here is a bit ambiguous but it was not introduced in this PR. I'll suggest we tackle it in another PR.

Reply:
My current feeling is `InteractiveWindowProvider.activeWindow` should be renamed to `focusedWindow` (as it's checking `activeNotebookEditor`) but I might need to verify all scenarios to see if that's valid.

Reply:
I think we should rename them to be more clear. Personally I'd rather have just one way to get active.